packages/api/cms-api/src/dam/files/file-validation.service.ts

```diff
@@ -16,10 +16,10 @@ export class FileValidationService {
         return error;
     }
 
-    async validateFileMetadata(file: FileUploadInput): Promise<undefined | string> {
-        //maximum file size
-        if (file.size > this.config.maxFileSize * 1024 * 1024) {
-            return "File is too large";
+    validateFileMetadata(file: Pick<FileUploadInput, "fieldname" | "originalname" | "encoding" | "mimetype">): undefined | string {
+        //maximum filename length
+        if (file.originalname.length > 255) {
```

Comment:
255 was chosen because it's also the max length on MacOS and Windows. So a filename longer than 255 characters was probably changed in JS for malicious reasons.